symfit/core/minimizers.py

```diff
@@ -382,79 +461,40 @@ def scipy_constraints(self, constraints):
+                    ))(list(p))
+            cons.append(con)
         cons = tuple(cons)
         return cons
 
 
-class BFGS(ScipyGradientMinimize):
+class BFGS(ScipyGradientMinimize, BoundedMinimizer):
     """
     Wrapper around :func:`scipy.optimize.minimize`'s BFGS algorithm.
     """
 
-class DifferentialEvolution(ScipyMinimize, GlobalMinimizer, BoundedMinimizer):
-    """
-    A wrapper around :func:`scipy.optimize.differential_evolution`.
-    """
-    @keywordonly(strategy='rand1bin', popsize=40, mutation=(0.423, 1.053),
-                 recombination=0.95, polish=False, init='latinhypercube')
-    def execute(self, **de_options):
-        ans = differential_evolution(self.list2kwargs(self.objective),
-                                     self.bounds,
-                                     **de_options)
-        return self._pack_output(ans)
-
-
-class SLSQP(ScipyConstrainedMinimize, GradientMinimizer, BoundedMinimizer):
+class SLSQP(ScipyGradientMinimize, ScipyConstrainedMinimize, BoundedMinimizer):
     """
     Wrapper around :func:`scipy.optimize.minimize`'s SLSQP algorithm.
     """
-    def __init__(self, *args, **kwargs):
-        super(SLSQP, self).__init__(*args, **kwargs)
-        # We have to break DRY because you cannot inherit from both
-        # ScipyConstrainedMinimize and ScipyGradientMinimize. So SLQSP is a
-        # special case. This is the same code as in ScipyGradientMinimize.
-        self.wrapped_jacobian = self.list2kwargs(self.wrapped_jacobian)
 
-    def execute(self, **minimize_options):
-        return super(SLSQP, self).execute(
-            bounds=self.bounds,
-            jacobian=self.wrapped_jacobian,
-            **minimize_options
-        )
-
-    def scipy_constraints(self, constraints):
-        """
-        Returns all constraints in a scipy compatible format.
-
-        :return: dict of scipy compatible constraints, including jacobian term.
-        """
-        # Take the normal scipy compatible constraints, and add jacobians.
-        scipy_constr = super(SLSQP, self).scipy_constraints(constraints)
-        for partialed_constraint, scipy_constraint in zip(constraints, scipy_constr):
-            partialed_kwargs = partialed_constraint.keywords
-            # In the case of the jacobian, first eval_jacobian of the
-            # constraint has to be partialed since that has not been done
-            # yet. Then it is made callable by keywords only, and finally
-            # the shape of the jacobian is made to match the number of
-            # unfixed parameters in the call, len(p).
-            scipy_constraint['jac'] = lambda p, c: self.resize_jac(
-                    self.list2kwargs(
-                        partial(c.func.eval_jacobian, **partialed_kwargs)
-                    )
-                )(list(p))
-        return scipy_constr
-
-
-class COBYLA(ScipyConstrainedMinimize):
+class COBYLA(ScipyConstrainedMinimize, BoundedMinimizer):
```

Comment:
`COBYLA` does not support bounds according to the docs. https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html#optimize-minimize-cobyla